cmd/run.go

```diff
@@ -61,6 +90,17 @@ func (h *Handler) Run(ctx context.Context, req *entity.CommandRequest) error {
 
 	printLooksGood()
 
+	if isEphemeral {
```

Comment:
Should we worry about ephemeral environments never being deleted? If the command errors or sigkilled will the ephemeral environment be removed?

Reply:
Probs not, but how should we listen to this one?